cli/util/errors.go

```diff
@@ -0,0 +1,7 @@
+package util
+
+import "errors"
+
+var (
+	ErrCmdAbort = errors.New("abort")
```

Comment:
Simple "abort" reminds me about sigabrt and `abort()` call. Let' use "aborted by user" error text.

Reply:
Fixed.